app/src/main/java/edu/grinnell/kdic/RadioService.java

```diff
@@ -87,7 +84,6 @@ public int onStartCommand(Intent intent, int flags, int startId) {
         if (intent != null && intent.getAction() != null) {
             switch (intent.getAction()) {
                 case Constants.ACTION_STOP_RADIO_SERVICE:
```

Comment:
Use static imports for  the constants here. (including the all the `AudioManager` constants)